source/Client/Client.csproj

```diff
@@ -14,6 +14,6 @@
     <PackageProjectUrl>https://github.com/OctopusDeploy/JiraIntegration</PackageProjectUrl>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Octopus.Client" Version="7.2.0" />
+    <PackageReference Include="Octopus.Server.Client" Version="11.2.3307-build-ilmerged-and-not" />
```

Comment:
```suggestion
    <PackageReference Include="Octopus.Server.Client" Version="11.2.3319" />
```

Reply:
Good catch. I have force-pushed updates lined up to do this to reduce commit noise once it merges. I'm working through those at the moment.